pkg/services/auth/authtoken/session_cleanup.go

```diff
@@ -0,0 +1,54 @@
+package authtoken
+
+import (
+	"context"
+	"time"
+)
+
+func (srv *UserAuthTokenServiceImpl) Run(ctx context.Context) error {
+	if srv.Cfg.ExpiredTokensCleanupIntervalDays <= 0 {
+		srv.log.Debug("cleanup of expired auth tokens are disabled")
+		return nil
+	}
+
+	jobInterval := time.Duration(srv.Cfg.ExpiredTokensCleanupIntervalDays) * 24 * time.Hour
```

Comment:
As you mentioned during our last convo this requires the application to be running for 7 days until we clean up old sessions. I think we can run this at startup (since it's not blocking the server) and then every hour. 

I dont see any reason why someone would configure how often this runs?

Reply:
Changed it so cleanup is run when Grafana starts. Change interval to 1 hour,